ksql-functional-tests/src/test/resources/query-validation-tests/joins.json

```diff
@@ -1285,6 +1427,9 @@
         "There are explicit tests for JSON and AVRO, rather than making use of the 'format's node,",
         "as the internal topic schema differs between the formats"
       ],
+      "versions": {
```

Comment:
These tests test legacy behaviour, so I've pinned them to legacy versions and _not_ added a new test for new versions as doing so wouldn't make sense.